samples/wifi/scan/prj.conf

```diff
@@ -7,6 +7,7 @@ CONFIG_WIFI=y
 CONFIG_WIFI_NRF700X=y
 CONFIG_NET_L2_WIFI_MGMT=y
 CONFIG_HEAP_MEM_POOL_SIZE=25000
+CONFIG_WPA_SUPP=n
```

Comment:
I think mqtt_multi_service and location sample WiFi scan-only overlays will need this change as well, as well as any other samples that have scan-only overlays for nrf9160. Or am I mistaken?

Reply:
Oh, my bad